zebROS_ws/src/controllers_2023/src/four_bar_controller.cpp

```diff
@@ -0,0 +1,461 @@
+//HPP CONTENTS INTO CPP
+#ifndef FOUR_BAR_CONTROLLER_2023
+#define FOUR_BAR_CONTROLLER_2023
+
+//added _2023 to the two lines above ^
+
+#include <ros/ros.h>
+#include <hardware_interface/joint_command_interface.h>
+#include <realtime_tools/realtime_buffer.h>
+#include <controller_interface/multi_interface_controller.h>
+#include <talon_controllers/talon_controller_interface.h> // "
+#include <pluginlib/class_list_macros.h> //to compile as a controller
+#include "controllers_2023_msgs/FourBarSrv.h"
+
+#include "ddynamic_reconfigure/ddynamic_reconfigure.h"
+
+namespace four_bar_controller_2023
+{
+
+class FourBarCommand_2023
```

Comment:
If we're just passing a single double, all of this could probably be cleaned up to be a `std::atomic<double>`.  That would let you remove the FourBarCommand, and turn the realtime_buffer into just an `std::atomic<double>`.  

On the other hand, if we'd expect to have to pass more info in the future, ignore me

Reply:
good idea, thanks!